docs/audit-log-filter/filter-audit-log-filter-files.md

```diff
@@ -0,0 +1,22 @@
+# Filter the Audit Log Filter logs
```

Comment:
This is the work in progress for now, right?

Reply:
Updated.